pkg/app/web/src/components/application-detail.tsx

```diff
@@ -63,10 +61,12 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.text.secondary,
     marginLeft: theme.spacing(1),
   },
-  syncStatusText: {
-    marginLeft: theme.spacing(0.5),
+  AppSyncState: {
```

Comment:
nit, do we have conventions for CSS classes' naming, looks like others start with normal character (not capital) 👀

Reply:
Nice catch!